.gitignore

```diff
@@ -18,7 +18,10 @@
 # Ignore code coverage folder
 coverage/*
 
+config/docker/Dockerfile.*
```

Comment:
One more question, what's this? Does docker generate a lock file or something like that?